tests/Functional/LocalPhpServerTestCase.php

```diff
@@ -49,8 +49,12 @@ public static function startServer($wwwRoot = null, $port=8000)
     ];
 
     self::$server = new Process($cmd);
+    self::$server->setTimeout(15);
     self::$server->start();
-    sleep(5);
+
+    self::$server->waitUntil(function ($type, $output) {
```

Comment:
PHP server starts and writes to stderr - because its a long running process, we can't be sure its up until we find `started` in stderr. The symfony process handler will timeout otherwise and we'll have errors.